src/services/v1-indexer/getHistoricalRatesGCloud.ts

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import { getSentryTracker } from '../../init';
+import { SupportedChainId } from '../../types';
+import { getServiceUrl } from './urls';
+
+export type HistoricalRate = {
+  rate: number;
+  timestamp: number;
+};
+
+export const getFixedRatesGCloud = async (
+  chainId: SupportedChainId,
+  vammAddress: string,
+  startTimestamp: number,
+  endTimestamp: number,
+): Promise<HistoricalRate[]> => {
+  try {
+    const baseUrl = getServiceUrl('fixed-rates');
+    const url = `${baseUrl}/${chainId}/${vammAddress}/${startTimestamp}/${endTimestamp}`;
+    const res = await axios({
```

Comment:
you can transform this into axios.get<HistoricalRate[]>(url, {       withCredentials: false }); to add type safety of the res.data.

also improves readability imho.

Reply:
[ab441d4](https://github.com/Voltz-Protocol/v1-sdk/pull/294/commits/ab441d4d20c43630a1af18c0ca119dc476ca7676)